requirements.txt

```diff
@@ -1,6 +1,6 @@
 absl-py
 cirq-core~=1.0
-numpy~=1.16
-pybind11
+numpy>=1.26.0
+pybind11[global]
```

Comment:
What does the global signify?  Is that because we added platform dependent stuff in the other PRs?

Reply:
Is it worth a one-line comment to explain this?